tests/Makefile.am

```diff
@@ -3,9 +3,15 @@ EXTRA_DIST = readme.txt
 AM_CPPFLAGS = \
   -I$(top_srcdir)/include
 
-bin_PROGRAMS = rfxcodectest
+bin_PROGRAMS = rfxcodectest rfxencode
```

Comment:
I have doubts whether those binaries need to be installed. If not, `check_PROGRAMS` should be used.